server/form-pages/apply/access-and-healthcare/accessNeedsAdditionalAdjustments.test.ts

```diff
@@ -0,0 +1,132 @@
+import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../shared-examples'
+
+import AccessNeedsAdditionalAdjustments from './accessNeedsAdditionalAdjustments'
+
+import applicationFactory from '../../../testutils/factories/application'
+import personFactory from '../../../testutils/factories/person'
+import { Application, Person } from '../../../@types/shared'
+import { SessionDataError } from '../../../utils/errors'
+
+describe('AccessNeedsAdditionalAdjustments', () => {
+  let application: Application
+  let person: Person
+  const previousPage = 'previousPage'
+
+  beforeEach(() => {
+    person = personFactory.build({ name: 'John Wayne' })
+    application = applicationFactory.build({
+      person,
+      data: {
+        'access-and-healthcare': {
+          'access-needs': { additionalNeeds: ['mobility', 'visualImpairment', 'neurodivergentConditions'] },
+        },
+      },
+    })
+  })
+
+  describe('title', () => {
+    it('has the correct title', () => {
+      expect(new AccessNeedsAdditionalAdjustments({}, application, previousPage).title).toBe('Access needs')
+    })
+  })
+
+  describe('this', () => {
```

Comment:
Glad you raised this. Obviously a typo on the name of the 'describe' but we need to wrap these two `it`s in a describe otherwise they don't recieve the `application` in the beforeEach